### PR TITLE
Displays a message when wicked is not active

### PR DESCRIPTION
**Pull request by dgdavid**
Instead of just displaying the `Interfaces` tab with an empty list, now the `cockpit-wicked` package will warn that is not possible to work with wicked because it looks either, not active or nor installed.

| Checking | Service not active |
| - | - |
| Screenshot_2020-11-13 Networking - ytm 10 0 0 204 | Screenshot_2020-11-13 Networking - ytm 10 0 0 204(1) |

### CODE REVIEW

**Comment by dgdavid** [comment]
@lslezak, @imobachgs, @teclator help needed here :)

Especially with the text to display. Thanks in advance!

**Comment by teclator** [comment]
For me it looks good, and I'm now that was more or less what was agreed. Maybe would check the current network.sservice in order to decide if it is handled by wicked or not. In case that the network.service is a link to wicked.service then would permit to start it. But of course, it is a corner case as I would expect the network to be running...

File: src/lib/wicked/client.js [hunk 1]
```diff
@@ -131,6 +131,21 @@ class WickedClient {
...
 
+    /**
+     * Whether the wicked service is active or not
+     *
+     * @return {Promise.<boolean>} Promise that resolves to true if service is active or false if not
+     */
+    async isActive() {
+        // FIXME: not using "--value" option here for backward compatibility with systemd < 230
+        // https://lists.freedesktop.org/archives/systemd-devel/2016-May/036583.html
+        const command = 'systemctl show wicked -p ActiveState';
```
Comment (dgdavid) [inline src/lib/wicked/client.js:142]: As discussed in a meeting with @imobachgs and  @teclator , probably is better to use `systemctl --system is-active wicked` instead.

**Review by teclator — APPROVED** [review]
LGTM

**Comment by teclator** [comment]
still LGTM

**Pull request merged**